models/user_test.go

```diff
@@ -34,8 +34,10 @@ func TestUserManager_NewAccount(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if _, err := um.NewUserAccount(tt.args.userName, tt.args.password, tt.args.email); (err != nil) != tt.wantErr {
+			if usr, err := um.NewUserAccount(tt.args.userName, tt.args.password, tt.args.email); (err != nil) != tt.wantErr {
 				t.Fatalf("NewUserAccount err = %v, wantErr %v", err, tt.wantErr)
+			} else if usr != nil && usr.Organization != "" {
```

Comment:
Remove else since the if case panics.
Remove usr != nil since usr == nil should be a fail case too

Reply:
done